spec/models/petition_spec.rb

```diff
@@ -334,6 +334,22 @@
       end
     end
 
+    context 'awaiting_debate_date' do
+      before do
+        @p1 = FactoryGirl.create(:open_petition)
+        @p2 = FactoryGirl.create(:awaiting_debate_petition)
+        @p3 = FactoryGirl.create(:debated_petition)
+      end
+
+      it 'returns only petitions that reached the debate threshold' do
+        expect(Petition.awaiting_debate_date).to include(@p2)
+      end
+
+      it 'doesn\'t include petitions that has the debate date' do
+        expect(Petition.awaiting_debate_date).not_to include(@p3)
+      end
+    end
+
```

Comment:
Be nice to have a spec for the instance version of this ``at_threshold_for_debate?`` too.  Just for clarity (and to make sure it's symmetrical with the scope version).